view/frontend/web/js/view/payment/method-renderer/omise-offline-conveniencestore-method.js

```diff
@@ -2,60 +2,60 @@ define(
     [
         'jquery',
         'ko',
+        'Omise_Payment/js/view/payment/omise-base-method-renderer',
         'Magento_Checkout/js/view/payment/default',
         'Magento_Checkout/js/model/full-screen-loader',
         'Magento_Checkout/js/action/redirect-on-success',
         'Magento_Checkout/js/model/quote',
-        'Magento_Customer/js/view/customer',
-        'Magento_Checkout/js/model/error-processor',
-        'Magento_Catalog/js/price-utils',
-        'mage/validation',
+        'Magento_Catalog/js/price-utils'
```

Comment:
I think we want to remove this module as well?
Also for `jquery` and `fullScreenLoader`.

Reply:
Redirect on success can also go if/when placeOrder is moved out

Reply:
Yup, priceUtils can go

Reply:
👍